parlai/tasks/decanlp/agents.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+# Download and build the data if it does not exist.
+
+from parlai.core.teachers import DialogTeacher
+from .build import build
+import parlai.tasks.squad.agents as squad
+import parlai.tasks.iwslt14.agents as iwslt14
+import parlai.tasks.cnn_dm.agents as cnn_dm
+import parlai.tasks.multinli.agents as multinli
+import parlai.tasks.sst.agents as sst
+import parlai.tasks.qasrl.agents as qasrl
+import parlai.tasks.qazre.agents as qazre
+import parlai.tasks.woz.agents as woz
+import parlai.tasks.wikisql.agents as wikisql
+import parlai.tasks.mwsc.agents as mwsc
+
+
+class SquadTeacher(squad.DefaultTeacher):
+    pass
+
+
+class Iwslt14Teacher(iwslt14.DefaultTeacher):
+    pass
+
+
+class Cnn_DmTeacher(cnn_dm.DefaultTeacher):
+    pass
+
+
+class MultinliTeacher(multinli.DefaultTeacher):
+    pass
+
+
+class SstTeacher(sst.DefaultTeacher):
+    pass
+
+
+class QasrlTeacher(qasrl.DefaultTeacher):
+    pass
+
+
+class QazreTeacher(qazre.DefaultTeacher):
+    pass
+
+
+class WozTeacher(woz.DefaultTeacher):
+    pass
+
+
+class WikisqlTeacher(wikisql.DefaultTeacher):
+    pass
+
+
+class MwscTeacher(mwsc.DefaultTeacher):
+    pass
+
+
+class DecaNLPTeacher(DialogTeacher):
```

Comment:
This should actually be formatted like parlai/tasks/babi/agents.py's All10kTeacher so that each sub-teacher preserves its ID

Reply:
Deleted the file